src/tests/ftest/tags.py

```diff
@@ -0,0 +1,458 @@
+#!/usr/bin/env python3
+"""
+  (C) Copyright 2024 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import ast
+import os
+import re
+import sys
+from argparse import ArgumentParser, RawDescriptionHelpFormatter
+from collections import defaultdict
+from copy import deepcopy
+from pathlib import Path
+
+THIS_FILE = os.path.realpath(__file__)
+FTEST_DIR = os.path.dirname(THIS_FILE)
+ROOT_DIR = os.path.realpath(os.path.join(FTEST_DIR, '..', '..', '..'))
```

Comment:
ROOT_DIR of what?  The repo?

What if this file is installed as part of the rpm, what's the ROOT_DIR then?

Reply:
It was intended to be the root dir of the repo, but I went with a different approach. So it's not needed anymore

Reply:
Removed. Thanks